README.md

```diff
@@ -184,4 +184,64 @@ In this directory (you should see the `test` and `traffic_comparator` directorie
 
 ```
 pytest
-```
\ No newline at end of file
+```
+
+## Exploring Results in Jupyter Notebook/IPython
+
+Currently, comparison results can be reported in very high level reports (average latencies, percent of matching status codes), or in very low level detail (all line-by-line diffs output to a file). Neither of these formats are conducive to data exploration or visualization.
+
+A new option has been added: comparison results can be dumped to a sqlite database, and then read in using some provided functions in a Jupyter Notebook or IPython terminal, and explored and visualized there.
```

Comment:
Nit - this reads more like a commit message rather than documentation, particularly with the "command has been added" phrasing. Consider rewording this to something like:

Comparison results can also be dumped to a sqlite database, thereby allowing the comparison data to be visualized via a Jupyter Notebook or IPython terminal.